stable/traefik/values.yaml

```diff
@@ -88,6 +88,13 @@ ssl:
   auth: {}
     # basic:
     #   testuser: $apr1$JXRA7j2s$LpVns9vsme8FHN0r.aSt11
+  mtls:
```

Comment:
Can you make this optional (commented out) or with an empty object (as with `auth: {}` above?)

Reply:
I am happy to do this. However does not make sense to keep it the way it is with an enabled flag? This seems to be inline with the rest of the chart.